tests/test_sqs.py

```diff
@@ -85,6 +85,25 @@ def test_batch_tasks(sqs, queue_name):
     assert len(queue.get_raw_messages(0)) == 2
 
 
+def test_big_batches(sqs, queue_name):
```

Comment:
I suggest splitting this test into three smaller tests to follow more strictly the arrange-act-assert pattern.

- `test_batch_should_keep_messages_until_overflow`
- `test_batch_should_flush_on_add`
- `test_batch_should_flush_on_exit`